opta/mattermost-aws.yaml

```diff
@@ -0,0 +1,50 @@
+name: mattermost-team
```

Comment:
This file is more suited in docs.mattermost.com (https://github.com/mattermost/docs). Feel free to send a PR over there and @cwarnermm will guide you.

Reply:
Awesome, I'll move the changes to there

Reply:
@agnivade actually, this is a copy of what is on our official examples here: https://github.com/run-x/opta/tree/main/examples/mattermost-team

I made a copy as I figured you may want to maintain your own copy (although it would lead to duplication), but while we're on the topic, perhaps you guys are ok referencing the yaml we are maintaining?

Reply:
I'll defer to @cwarnermm here.